api/pom.xml

```diff
@@ -13,79 +13,56 @@
 	<description>API project for patientdashboardui</description>
 
 	<dependencies>
-	
-		<!-- 
-		Add other dependencies from parent's pom:
+
 		<dependency>
-			<groupId>org.other.library</groupId> 
-			<artifactId>library-name</artifactId>
-		</dependency> 
-		-->
+			<groupId>org.openmrs.api</groupId>
+			<artifactId>openmrs-api</artifactId>
+			<type>jar</type>
+		</dependency>
 
-		
-				
-		<!-- Begin OpenMRS modules -->
-				
 		<dependency>
-		
 			<groupId>org.openmrs.module</groupId>
-									
-			<artifactId>uiframework-api</artifactId>
-															
+			<artifactId>hospitalcore-api</artifactId>
 		</dependency>
-				
 		<dependency>
-		
 			<groupId>org.openmrs.module</groupId>
-									
-			<artifactId>uilibrary-api</artifactId>
-															
+			<artifactId>ehrinventory-api</artifactId>
 		</dependency>
-		
+
 		<dependency>
 			<groupId>org.openmrs.module</groupId>
-			<artifactId>hospitalcore-api</artifactId>

```

Comment:
So hospital core is NOT required? I thought it was a matter of replacing the version only